src/funding/card/config.jsx

```diff
@@ -122,10 +122,20 @@ export function getCardConfig() : FundingSourceConfig {
         },
 
         Label: ({ logo, locale, content, custom }) => {
-            if (custom && custom.label) {
+            if (custom) {
+                const validLabels = {
+                    checkout: 'Checkout'
+                };
+
+                let label = 'Checkout';
+
+                if (custom.label && typeof custom.label === 'string' && validLabels[custom.label.toLowerCase()]) {
```

Comment:
can flow catch type of label or does this come from external source?

Reply:
external source